### ISSUE TITLE
[WebToolsE2E][Aspire][MacOS]Create An Aspire Starter App project and click Trace's ID in Structured page and click the '-' button to the left of trace, it isn't clickable  in Safari browser.

### ISSUE THREAD

**Issue by v-polly** [dotnet/aspire#2675]
**REGRESSION INFO:** This is a new feature on Aspire 8.0.0-preview.4.24129.7

**INSTALL STEPS**

1.Clean machine: MacOS Ventura[13.6.4]
2.Install VS Code[1.87.0] and C# Dev Kit[1.4.28]
3.Install SDK 8.0.202
4.Install Aspire 8.0 P4 workload version 8.0.0-preview.4.24129.7
5.Apply NuGet Feeds

**REPRO STEPS**

1.Open CMD, creating an Aspire Starter App using following commands:
```
dotnet new aspire-starter -o aspirestart1
cd aspirestart1
code .
```
2.Right-click xxx.AppHost > Debug > Start New Instance
3.Click the endpoints of apiservice and webfrontend respectively on the web page at the same time, then go to 'Structured' page > click Trace's ID to enter
4.Click the '-' button to the left of trace

**ACTUAL:**
There was no pop-up message next to the trace line.
![image](https://github.com/dotnet/aspire/assets/88425944/99885812-2023-47e3-8970-69fa8ff74f92)

**NOTE:**
1.Safari version is 17.3.1(18617.2.4.11.12, 18617).
2.It is not repro with Edge / Firefox / Google Chrome.

**EXPECTED**
The '-' button can be clicked and changed to '+'.
![image](https://github.com/dotnet/aspire/assets/88425944/5da13332-aad8-43c5-8f32-c68ac78804ad)
![image](https://github.com/dotnet/aspire/assets/88425944/d9721651-a08b-4352-bb52-2174d1515093)

**Comment by adamint** [comment]
Duplicate of #2272

**Comment by v-polly** [comment]
This issue was fixed and not reproduce with Safari version 17.4.1 (19618.1.15.11.14)
